### PR TITLE
Upgraded iterm2 to '3.4.9beta1'.

### PR DESCRIPTION
**Pull request by vraravam**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-versions/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/iterm2-beta.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 cask "iterm2-beta" do
-  # NOTE: "2" is not a version number, but an intrinsic part of the product name
-  version "3.4.7beta2"
-  sha256 "4e4cf559c7cc150b4ef929e17d3af52b8f7660bfd0c340c57bd2d423cdf06ca7"
+  # NOTE: "1" is not a version number, but an intrinsic part of the product name
```
Comment (carlocab) [inline Casks/iterm2-beta.rb:2]: Why change this? The original note was referring to the `2` in `iterm2`. Now it's referring to nothing at all.

**Comment by carlocab** [comment]
Version bump merged in #11179.

File: Casks/iterm2-beta.rb (hunk 1 above)
Comment (vraravam) [reply Casks/iterm2-beta.rb:2]: imp - the `Note` is in the wrong line in that case. It was right above the `version` line - and thus, its natural to assume the note refers to that line.

Comment (carlocab) [reply Casks/iterm2-beta.rb:2]: It's fine where it is -- the note is relevant to both the line above it and the line below it. It says that you shouldn't try to move the `2` in `iterm2` from the cask name to `version`. Putting it elsewhere is probably wrong.

**Pull request closed without merging**